crates/chia-consensus/src/multiprocess_validation.rs

```diff
@@ -0,0 +1,467 @@
+        let mut result = msg.to_vec();
+        result.extend(
+            [
+                test_coin.coin_id().as_slice(),
+                TEST_CONSTANTS.agg_sig_me_additional_data.as_slice(),
+            ]
+            .concat(),
+        );
+        let sig = sign(&sk, result.as_slice());
+        let coin_spends: Vec<CoinSpend> = vec![spend];
+        let spend_bundle = SpendBundle {
+            coin_spends: coin_spends,
+            aggregated_signature: sig,
+        };
+        let _result = validate_clvm_and_signature(
+            &spend_bundle,
+            TEST_CONSTANTS.max_block_cost_clvm,
+            &TEST_CONSTANTS,
+            1,
+            Arc::new(Mutex::new(BlsCache::default())),
+        )
+        .expect("SpendBundle should be valid for this test");
+    }
+
+    #[test]
+    fn test_validate_aggsig_parent_puzzle() {
+        let sk_hex = "52d75c4707e39595b27314547f9723e5530c01198af3fc5849d9a7af65631efb";
+        let sk = SecretKey::from_bytes(&<[u8; 32]>::from_hex(sk_hex).unwrap()).unwrap();
+        //let pk: PublicKey = sk.public_key(); //0x997cc43ed8788f841fcf3071f6f212b89ba494b6ebaf1bda88c3f9de9d968a61f3b7284a5ee13889399ca71a026549a2
+        // panic!("{:?}", pk);
+
+        let full_puz = Bytes32::new(tree_hash_atom(&[1_u8]).to_bytes());
+        let test_coin = Coin::new(
+            Bytes32::new(hex!(
+                "4444444444444444444444444444444444444444444444444444444444444444"
+            )),
+            full_puz,
+            1,
+        );
+
+        let solution = Bytes::new(hex!("ffff30ffb0997cc43ed8788f841fcf3071f6f212b89ba494b6ebaf1bda88c3f9de9d968a61f3b7284a5ee13889399ca71a026549a2ff8568656c6c6f8080").to_vec());
+        // ((48 0x997cc43ed8788f841fcf3071f6f212b89ba494b6ebaf1bda88c3f9de9d968a61f3b7284a5ee13889399ca71a026549a2 "hello"))
+
+        let spend = CoinSpend::new(
+            test_coin,
+            Program::new(vec![1_u8].into()),
+            Program::new(solution),
+        );
+        let msg = b"hello";
+        let mut result = msg.to_vec();
+        result.extend(
+            [
+                test_coin.parent_coin_info.as_slice(),
+                test_coin.puzzle_hash.as_slice(),
+                TEST_CONSTANTS
+                    .agg_sig_parent_puzzle_additional_data
+                    .as_slice(),
+            ]
+            .concat(),
+        );
+        let sig = sign(&sk, result.as_slice());
+        let coin_spends: Vec<CoinSpend> = vec![spend];
+        let spend_bundle = SpendBundle {
+            coin_spends: coin_spends,
+            aggregated_signature: sig,
+        };
+        let result = validate_clvm_and_signature(
+            &spend_bundle,
+            TEST_CONSTANTS.max_block_cost_clvm,
+            &TEST_CONSTANTS,
+            TEST_CONSTANTS.hard_fork_height - 1,
+            Arc::new(Mutex::new(BlsCache::default())),
+        );
+        if let Ok(_) = result {
+            panic!("height too low!")
+        };
```

Comment:
I could be wrong, but it looks like this would be simpler to just:

```suggestion
        assert!_eq(validate_clvm_and_signature(
            &spend_bundle,
            TEST_CONSTANTS.max_block_cost_clvm,
            &TEST_CONSTANTS,
            TEST_CONSTANTS.hard_fork_height - 1,
            Arc::new(Mutex::new(BlsCache::default())),
        ).unwrap_err(), <expected_error>);
```

Reply:
This test has been removed now anyway as we removed the fork height